src/types/EtherspotUi.ts

```diff
@@ -60,6 +60,7 @@ export interface IEtherspotTokenTransferTransaction {
 export interface ISmartWalletAddress {
   chainId: number;
   address: string;
+  name: string,
```

Comment:
to not confuse with ENS name or any other way of naming address
```suggestion
  chainName: string,
```

Reply:
ideally maybe store `chainName` and `chainId` within `chain`? wdyt

Reply:
Totally agree, will change this